expo/gbe/views/review_bid_view.py

```diff
@@ -0,0 +1,89 @@
+from django.views.generic import View
+from django.contrib.auth.decorators import login_required
+from django.shortcuts import (
+    render,
+    get_object_or_404,
+)
+from django.http import HttpResponseRedirect
+from django.core.urlresolvers import reverse
+from expo.gbe_logging import log_func
+from gbe.forms import (
+    BidEvaluationForm,
+    BidStateChangeForm,
+    PersonaForm,
+    ClassBidForm,
+    ParticipantForm,
+
+)
+from gbe.models import (
+    Class,
+    BidEvaluation,
+)
+from gbe.functions import (
+    validate_perms,
+    get_conf,
+)
+
+
+class ReviewBidView(View):
+
+    def create_action_form(self, object):
+        self.actionform = BidStateChangeForm(instance=self.object)
+        self.actionURL = reverse('class_changestate',
```

Comment:
Why does this give the value for 'class_changestate' - seems like it should do something universtal or it should do whatever the subclass needs...